bob/Cargo.toml

```diff
@@ -36,7 +36,7 @@ serde_yaml = "0.8"
 stopwatch = "0.0.7"
 termion = "1.5"
 thiserror = "1.0"
-sysinfo = "0.21"
+sysinfo = "0.21.2"
```

Comment:
Why this was updated?

Reply:
It was updated for some reason during feature development, don't remember why